todo_app/viewmodel.py

```diff
@@ -0,0 +1,35 @@
+from todo_app.data.todo_item import Item
+
+
+class ViewModel:
+    def __init__(self, items: list[Item]):
+        self._items = items
+ 
+    @property
+    def items(self):
+        return self._items
+    
+    @property
+    def todo_items(self):
+        all_items = self._items
+
+        output = []
+
+        for item in all_items:
+            if item.status == "To Do":
+                output.append(item)
+
+        return output
```

Comment:
Not something I'm recommending to change, but I do think worth seeing a comparison with a "list comprehension" approach, which is much more concise but arguably harder to understand!
```python
    @property
    def todo_items(self):
        all_items = self._items
        return [item for item in all_items if item.status == "Done"]
```